paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/common/Element.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.paymentsheet.elements.common
+
+import kotlinx.coroutines.flow.Flow
+
+interface Element {
```

Comment:
Can you add KDoc comments to the interfaces and their fields? Also on DropdownConfig.